.circleci/config.yml

```diff
@@ -186,17 +180,17 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - deps8-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-            - deps8-
-      - run: yarn install --immutable
-      - save_cache:
-          key: deps8-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
-          paths:
-            - node_modules
-            - .yarn/cache
+            - deps-{{ .Branch }}-{{ checksum ".pnp.cjs" }}
+            - deps-{{ .Branch }}-
```

Comment:
What is line 184 for?

Reply:
To allow for a partial restore if there isn't an exact hit. It'll then run the new install and save the updated cache entry with the new key.